src/color.js

```diff
@@ -172,12 +177,47 @@ export default class Color {
 		}, 0));
 	}
 
+	/**
+	 * HyAB distance of colors in an Lab-like color space.
+	 * Best used for large distances. Claims to outperform even DE2000 at that.
+	 * Said to be okay for small distances.
+	 * 
+	 * @see http://markfairchild.org/PDFs/PAP40.pdf
+	 * @param {Color} color   - The other color.
+	 * @param {string} space  - Color space to run in.
+	 */
+	distanceHyAB (color, space = "lab") {
```

Comment:
This should definitely be a separate module.

I wonder if we should add an `options` argument to `distance()` so this can be invoked that way.

Reply:
The similarity in argument normalisation does make me think that way.

Reply:
We can keep the current signature as well, branch if the second argument is an object and treat it as an options object (`{space, method}`). Plugins can add methods, like they do for `deltaE`.
We'd probably want to throw if HyAB distance is used with a non-labish space.

Reply:
Yes, this should be a module under [`src/deltaE`](https://github.com/LeaVerou/color.js/tree/master/src/deltaE)